fission/src/ui/modals/spawning/ManageAssembliesModal.tsx

```diff
@@ -20,7 +20,9 @@ const AssemblyCard: React.FC<AssemblyCardProps> = ({ id, update }) => {
             <Label className="text-wrap break-all">{id}</Label>
             <Button
                 value="Delete"
+                // prettier-ignore
                 onClick={() => {
+                    (World.SceneRenderer.sceneObjects.get(id) as MirabufSceneObject).DisableTransformControls()
```

Comment:
I would move this to be included in the Dispose function of `MirabufSceneObject`